src/Query/Filter/DateField.ts

```diff
@@ -52,9 +52,9 @@ export abstract class DateField extends Field {
 
         const fieldNameKeywordDate = Field.getMatch(this.filterRegExp(), line);
         if (fieldNameKeywordDate !== null) {
-            const keywordAndDateString = fieldNameKeywordDate[1]; // Will contain the whole line except the field name
-            const fieldKeyword = fieldNameKeywordDate[2]; // Will be 'before', 'after', 'in', 'on' or undefined
-            const fieldDateString = fieldNameKeywordDate[3]; // Will contain the remainder of the instruction
+            const keywordAndDateString = fieldNameKeywordDate[1]; // The whole line except the field name
+            const fieldKeyword = fieldNameKeywordDate[2]; // '(on|in or )before', '(on|in or )after', 'in', 'on', or undefined
```

Comment:
What do the `()` mean in this line? I don't understand this comment any more...

Reply:
Yup, indeed, fixed (reused the better order from the docs)